Tools/clinic/clinic.py

```diff
@@ -1264,27 +1267,8 @@ def parser_body(
             parser_prototype = self.PARSER_PROTOTYPE_VARARGS
             parser_definition = parser_body(parser_prototype, '    {option_group_parsing}')
 
-        elif (not requires_defining_class and pos_only == len(parameters) and
-              not pseudo_args and limited_capi):
-            # positional-only for the limited C API
```

Comment:
It is no longer needed, because there is other `PyArg_ParseTuple()` below which is used if inlining the parsing code failed. `PyArg_UnpackTuple()` no longer used, because inlining is always successful in this simple case.